public/js/report.js

```diff
@@ -259,7 +259,17 @@ var Report = React.createClass({
     },
 
     atLeastTwoHits: function () {
-        return this.state.queries.some(query => query.hits.length > 1);
+        var hit_arr = [];
+        var hit_num = 0;
+        _.each(this.state.queries, (function (obj) {
+            if (obj.hits.length > 0) {
+                hit_arr.push(obj.hits.length);
+            }
+        }));
+        for (var i = 0; i < hit_arr.length; i++) {
+            hit_num += hit_arr[i];
+        }
+        return (this.state.queries.length > 0 && hit_num > 1);
```

Comment:
There will always be at least one query. The first check is unnecessary

Reply:
Makes sense as well, thanks for pointing this out